oslabs-beta/Kubiquity
# Run linter

WHAT DOES THIS PR DO?

- Ran the linter across the entire workspace

HOW TO TEST THIS PR:

1. Run `gh pr checkout 44 && npm install && npm run dev`
2. Ensure that the app is still working
